pkg/generate/app/cmd/template.go

```diff
@@ -53,7 +53,10 @@ func formatString(out io.Writer, tab, s string) {
 	labelVals := strings.Split(s, "\n")
 
 	for _, lval := range labelVals {
-		fmt.Fprintf(out, fmt.Sprintf("%s%s\n", tab, lval))
+		s := fmt.Sprintf("%s%s\n", tab, lval)
+		if len(strings.TrimSpace(s)) > 0 {
+			fmt.Fprintf(out, fmt.Sprintf("%s%s\n", tab, lval))
+		}
```

Comment:
Seems reasonable to me @bparees 

Researching how the existing implementation got where it was, @juanvallejo added the `formatString` method in https://github.com/openshift/origin/pull/12724 as part of addressing https://bugzilla.redhat.com/show_bug.cgi?id=1370104 for another look and feel complaint against output from the CLI.

Far as I can tell from the comments in those 2 items, there was no explicit intent to strip out newlines from things like "foo\n\nbar\n" ... does that sound right to you @juanvallejo ?